src/solvers/prop/prop_conv_solver.h

```diff
@@ -124,10 +127,24 @@ class prop_conv_solvert : public conflict_providert,
 
   virtual void ignoring(const exprt &expr);
 
-  // deliberately protected now to protect lower-level API
+  /// Actually adds the constraints to `prop`
+  void do_set_to(const exprt &expr, bool value);
+
   propt &prop;
 
   messaget log;
+
+  static const char *context_prefix;
```

Comment:
Can we just initialise it right here rather than in the implementation file?

Reply:
What would be the advantage of that?

Reply:
Fewer lines of code and easier to locate. But clearly a nit-pick and all up to you.